src/main/java/it/gov/pagopa/payhub/activities/connector/sendnotification/SendServiceImpl.java

```diff
@@ -0,0 +1,21 @@
+package it.gov.pagopa.payhub.activities.connector.sendnotification;
+
+import it.gov.pagopa.payhub.activities.connector.auth.AuthnService;
+import it.gov.pagopa.payhub.activities.connector.sendnotification.client.SendClient;
+import org.springframework.stereotype.Service;
+
+@Service
```

Comment:
always use @Lazy on business @Service in the repo activites
```suggestion
@Lazy
@Service
```